Misc/NEWS.d/next/Core_and_Builtins/2025-06-16-18-16-41.gh-issue-135575._N5rTH.rst

```diff
@@ -0,0 +1 @@
+Improved performance of ``MutableMapping.update``.
```

Comment:
```suggestion
Improved performance of :meth:`MutableMapping.update`.
```

Reply:
I actually tried this one first, but CI complained that it cannot find the RST-reference.

https://github.com/python/cpython/compare/b78241be0ce1147a9b79b50edf2b8ddb95852f37..d6f4c823eaaa30642984f3a30e02f26c00112f23